docs/architecture.md

```diff
@@ -56,24 +56,36 @@ When building new client features, there are a few things worth noting about the
 > **Note**: Some folders do not fully comply with this design -- there is a cleanup effort underway -- do not let that dissuade you from following the correct architecture choices noted down below when adding new features / code.
 
 - `/src/api` -- The current API
+    - Allowed imports: `/src/utilities`
```

Comment:
Completely unrelated to the content, but why are we using 4 spaces to list indent? Should it be 2 by default? (I just realize that we use 4 for the whole document)

Reply:
It's whatever I did originally... it's 4 I thought for GitHub markdown and just always do 4 for matter what I'm writing (comments, descriptions, markdown, etc).

Reply:
Good point! I'll run prettier on the file